src/badge/Badge.tsx

```diff
@@ -12,6 +13,8 @@ const Badge = ({ className, children, ariaLabel }: BadgeProps) => {
   if (isEmpty(children)) {
     return null
   }
+  const value = useContext(MediaSizeContext)
+  console.log(value) // ouput correctly large / small
```

Comment:
maybe `console.info` instead

Reply:
Yes I'm cleaning the PR I'll be removing it, it was just for me :)